pkg/status/health/health.go

```diff
@@ -16,7 +16,7 @@ var bufferSize = 2
 
 // Handle holds the token and the channel for components to use
 type Handle struct {
-	C <-chan struct{}
+	C <-chan time.Time
```

Comment:
I was wondering if it could be interesting to define a struct for the token that contains the `time.Time`. Like this we can imagine be able to extended later the token.
WDYT

```golang
type Token struct {
    time.Time
}

func NewToken() {
   return Token{Time: time.Now()}
}

Reply:
Good idea !
Done: https://github.com/DataDog/datadog-agent/pull/7829/commits/7ace55ecf174b2e97f65198cc7d3c1047d5078d6